apisix/http/router/radixtree_host_uri.lua

```diff
@@ -94,7 +94,14 @@ local function create_radixtree_router(routes)
     host_router = nil
 
     for _, route in ipairs(routes or {}) do
+        local status = core.table.try_read_attr(route, "value", "status")
+        if status and status ~= 1 then
```

Comment:
magic number is tricky, what about using literal string here?

Reply:
just follow this
https://github.com/apache/apisix/blob/9e3c34a0103b88d340285017e34021cbc1f0fde8/apisix/schema_def.lua#L630-L635

Reply:
add comments?